about/introduction.rst

```diff
@@ -1,14 +1,17 @@
 Introduction
 ============
 
-Our goal is to create a copylefted libre mobile operating system. One you can use, study, change and share; with all.
+Our goal is to create a copylefted libre mobile operating system. One you can use, study, change and share; with all of the code released under the GNU General Public License, and anyone can inspect or modify the source code of any program that is included.
 
 About UBports
 -------------
 
-The project was founded by Marius Gripsgard in 2015 and in its infancy a place where developers could share ideas and educate each other in hopes of bringing the Ubuntu Touch platform to more mobile devices.
+Marius Gripsgard founded the project in 2015. In its infancy, developers could share ideas and educate each other to bring the Ubuntu Touch platform to more mobile devices. It now boasts a very active Linux community that brings along professional app developers to help improve the operating system for even more consumers.
```

Comment:
Would that last sentence not fit better in the last paragraph? This way it does mix history and present information.

...formalised as a charitable foundation. It now brings along professional app developers to help improve the operating system for even more consumers.